modules/KalturaSupport/components/dualScreen/dualScreen.js

```diff
@@ -621,7 +637,133 @@
 					at: location[0]+location[1],
 					of: $( this.getPlayer().getInterface() )
 				});
-			}
+			},
+			getKClient: function () {
```

Comment:
@OrenMe are you referring to the function 'getKClient'? This is leftover from initial implementation. I will remove it.